Vim.md

```diff
@@ -12,39 +12,39 @@ require the user to remove their hands from the home row.
 
 ### Getting started with Vim
 
-To start vim without opening a specific file simply type `vim` on the terminal.
-To open a file, write `vim filename`, in the case of this file, one would type
-`vim vim.md`, like you likely already have.
+To start Vim without opening a specific file simply type `Vim` on the terminal.
+To open a file, write `Vim filename`, in the case of this file, one would type
+`Vim Vim.md`, like you likely already have.
```

Comment:
I'm not sure vim should be capitalized when referring to launching it from the terminal.

Reply:
whoops, nice catch

Reply:
I'm thinking line 15 and 17 too